fr/integrations/plugin-packs/procedures/cloud-aws-ec2.md

```diff
@@ -3,102 +3,263 @@ id: cloud-aws-ec2
+<!--DOCUSAURUS_CODE_TABS-->
 
-  - Ec2-Cpu-Credit
-  - Ec2-Cpu-Usage
-  - Ec2-Diskio
-  - Ec2-Network
-  - Ec2-Status
+<!--Cloud-Aws-Ec2-Asg-&-Cloud-Aws-Ec2-Instance-->
 
-#### For an instance
+| Mandatory   | Nom             | Description                                                |
+| :---------- | :-------------- | :--------------------------------------------------------- |
+| X           | AWSINSTANCENAME | Name of the instance you want to monitor                   |
+| X           | AWSINSTANCETYPE | Type of instance to check ('instance' or 'cluster')        |
+
+<!--Cloud-Aws-Ec2-Spot-Fleet-Request-->
+
+| Mandatory   | Nom                | Description                                             |
+| :---------- | :----------------- | :------------------------------------------------------ |
+| X           | SPOTFLEETREQUESTID | Spot Fleet Request identifier. (e.g: sfr-abcd1234)      |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## FAQ
+
+### How can I test it through the CLI and what is the meaning of the command_line parameters ?
+
+A partir du moment ou le Plugin est installé, vous pouvez tester celui-ci directement depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
+(Certaines options, comme par exemple ```--name``` doivent être ajustées en fonction du contexte):
+
+```bash
+/usr/lib/centreon/plugins//centreon_aws_ec2_api.pl
+	--plugin=cloud::aws::ec2::plugin
+	--mode=cpu
+	--custommode='awscli'
+	--aws-secret-key='***'
+	--aws-access-key='AKIA5EDPTASPNBK5EMTM'
+	--region='eu-west-1'
+	--type='asg'
+	--name='centreon-front'
+	--filter-metric='Utilization'
+	--statistic='average'
+	--timeframe='600'
+	--period='60'
+	--warning-cpu-utilization='80'
+	--critical-cpu-utilization='90'
+```
+
+La commande produit le message de sortie ci-dessous:
+
+```bash 	
+OK: Asg 'centreon-front' Statistic 'Average' Metrics CPU Utilization: 35.81 | 'centreon-front~average#ec2.cpu.utilization.percentage'=35.81;80;90;;
+```
+
+Cette commande supervise la consommation CPU (```--mode=cpu```) sur le groupe d'Auto Scaling *centreon-front* (```--name='centreon-front' --type='asg'```). 
+Ce groupe est rattaché à la région 'eu-west-1' d'AWS (```--region='eu-west-1'```).
+
+La métrique obtenue est une moyenne de valeurs (```--statistic='average'```) sur un intervalle de 10 minutes / 600 secondes  (```--timeframe='600'```) avec un point par minute / 60 secondes (```--period='60'```).
+
+Une alerte WARNING sera déclenchée lorsque la consommation CPU sera supérieure à 80M, et CRITICAL si elle est supérieur à 90%.
+
+Pour chaque mode, les options disponibles peuvent être consultée via l'option ```--help```:
+
+```bash
+/usr/lib/centreon/plugins/centreon_aws_ec2_api.pl --plugin=cloud::aws::ec2::plugin --mode=cpu --help
+```
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+### UNKNOWN: No metrics. Check your options or use --zeroed option to set 0 on undefined values
```

Comment:
```suggestion
### J'obtiens le message d'erreur suivant:  ```UNKNOWN: No metrics. Check your options or use --zeroed option to set 0 on undefined values```
```